From 675bf209dd8bde40637b362c8660f0298e11f2bc Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Thu, 8 Jun 2017 17:30:29 +0000
Subject: [PATCH 249/462] Mantis 0045407: extension: Transmit Billing Data
 records to a configured WEB Service. Mantis 0046042: extension: CR 030 PNR
 Reporting Mantis 0045269: extension: Report Application Session- and
 Killtimeouts in billing files

added Jars:
JarBarcodePartser
JarLoggingClientAvinor
JarAvinor
JarMipsSoapTools

added PlatformDirList:
Soap-Tools
LoggingClientAvinor

trim DirList to allow splitting the PlatformDirList in multiple lines


git-svn-id: svn://localhost/SelfServiceBuild/trunk@388 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/java/AntProperties.xml |  6 ++++-
 SelfServiceBuild/Build/java/build.xml         | 25 +++++++++++++++++--
 2 files changed, 28 insertions(+), 3 deletions(-)

diff --git a/SelfServiceBuild/Build/java/AntProperties.xml b/SelfServiceBuild/Build/java/AntProperties.xml
index e081b38..01d40bc 100644
--- a/SelfServiceBuild/Build/java/AntProperties.xml
+++ b/SelfServiceBuild/Build/java/AntProperties.xml
@@ -207,7 +207,8 @@
 <property name="JarBeanUtilsCore"           value="${DIR_GLOBALS}/ThirdParty/commons-beanutils-core/1.7.0/commons-beanutils-core-1.7.0.jar"/>
 <property name="JarXalan"                   value="${DIR_GLOBALS}/ThirdParty/xalan/2.7.1/xalan-2.7.1.jar"/>
 
-
+<!-- BarcodeParser -->
+<property name="JarBarcodeParser"           value="${DIR_GLOBALS}/ThirdParty/barcodeParserLib/1.2.0.0/barcodeParserLib-1.2.0.0.jar"/>
 
 <!-- JRE -->
 <property name="JarAppletPlugin"            value="${JAVA_HOME}/jre/lib/plugin.jar"/>
@@ -245,6 +246,9 @@
 <property name="JarMatac"                   value="${JarDir}/MATAC-XBEANS.jar"/>
 <property name="JarApplicationAdapter"      value="${JarDir}/ApplicationAdapter.jar"/>
 <property name="JarMassaiEventReporting"    value="${JarDir}/massai-eventreporting.jar"/>
+<property name="JarLoggingClientAvinor"     value="${JarDir}/LoggingClientAvinor.jar"/>
+<property name="JarAvinor"                  value="${JarDir}/avinor.jar"/>
+<property name="JarMipsSoapTools"           value="${JarDir}/mips-soap-tools.jar"/>
 
 <!-- slf4j -->
 <property name="JarSlf4jApi"                value="${DIR_GLOBALS}/ThirdParty/slf4j/1.7.21/slf4j-api-1.7.21.jar"/>
diff --git a/SelfServiceBuild/Build/java/build.xml b/SelfServiceBuild/Build/java/build.xml
index 270a5d6..02b8903 100644
--- a/SelfServiceBuild/Build/java/build.xml
+++ b/SelfServiceBuild/Build/java/build.xml
@@ -24,7 +24,28 @@
 
     <!-- Definition der Verzeichnisse mit den Java-Modulen ======= -->
     <property name="CommonDirList"   value="${CommonJavaDir}/OpenORB-1.3.1/OpenORB"/>
-    <property name="PlatformDirList" value="${PlatformJavaDir}/CUSS,${PlatformJavaDir}/Ingenico-Tempus,${PlatformJavaDir}/OPI.1.3,${PlatformJavaDir}/OPI_CCV20120201,${PlatformJavaDir}/Tools,${PlatformJavaDir}/MATAC-Client,${PlatformJavaDir}/CussIF,${PlatformJavaDir}/Config,${PlatformJavaDir}/icnTunnelClient,${PlatformJavaDir}/PlatformManager,${PlatformJavaDir}/ApplicationAdapter,${PlatformJavaDir}/AdminApplication,${PlatformJavaDir}/SSDMApplication,${PlatformJavaDir}/PrintService,${PlatformJavaDir}/CORBA-Settings,${PlatformJavaDir}/SOAP-Settings,${PlatformJavaDir}/CLAApplet,${PlatformJavaDir}/DSGEventReporting"/>
+    <property name="PlatformDirList" value="
+      ${PlatformJavaDir}/CUSS,
+      ${PlatformJavaDir}/Ingenico-Tempus,
+      ${PlatformJavaDir}/OPI.1.3,
+      ${PlatformJavaDir}/OPI_CCV20120201,
+      ${PlatformJavaDir}/Tools,
+      ${PlatformJavaDir}/MATAC-Client,
+      ${PlatformJavaDir}/CussIF,
+      ${PlatformJavaDir}/Config,
+      ${PlatformJavaDir}/icnTunnelClient,
+      ${PlatformJavaDir}/PlatformManager,
+      ${PlatformJavaDir}/Soap-Tools,
+      ${PlatformJavaDir}/LoggingClientAvinor,
+      ${PlatformJavaDir}/ApplicationAdapter,
+      ${PlatformJavaDir}/AdminApplication,
+      ${PlatformJavaDir}/SSDMApplication,
+      ${PlatformJavaDir}/PrintService,
+      ${PlatformJavaDir}/CORBA-Settings,
+      ${PlatformJavaDir}/SOAP-Settings,
+      ${PlatformJavaDir}/CLAApplet,
+      ${PlatformJavaDir}/DSGEventReporting"
+    />
     <property name="AblsDirList"     value="${AblsJavaDir}/KioskProxy,${AblsJavaDir}/TunnelServer"/>
     <property name="AppDirList"      value=""/>
 
@@ -41,7 +62,7 @@
         <echo message="ProductName    = (${ProductName})"/>
         <echo message="VendorName     = (${VendorName})"/>
 
-        <for param="CurrentDir" list="${DirList}">
+        <for param="CurrentDir" list="${DirList}" trim="true">
             <sequential>
                 <echo message="Calling compiler for directory [@{CurrentDir}]..."/>
                 <if>
-- 
2.41.0.windows.1

